internal/driver/mobile/app/app.go

```diff
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-//go:build freebsd || linux || darwin || windows
-// +build freebsd linux darwin windows
+//go:build freebsd || linux || darwin || windows || openbsd
+// +build freebsd linux darwin windows || openbsd
```

Comment:
This build tag does not look correct. You need to remove the `||`

Reply:
Oh my, good catch, thanks. Fixed.